README.md

```diff
@@ -102,7 +102,7 @@ Commits to this repo (including Pull Requests) should be made on the Develop bra
 
 Finally, update your cf-release deployment to enable the Routing API included in this release.
 
-1. If you don't already have one, create a file for overriding manifest properties of cf-release. In the context of manifest generation, we call this file a stub; you could name it `cf-boshlite-stub.yml`. Add the following properties to this file. When you re-generate the manifest, these values will override the defaults in the manifest.
+1. If you have a stub for overriding manifest properties of cf-release, add the following properties to this file. A default one is provided. When you re-generate the manifest, these values will override the defaults in the manifest.
```

Comment:
Hi Dies,

Is it ok if add a hyperlink to the "default one"? 
We can add if you want. Let us know.

Cheers
Mark && Shash